estate/models/estate_property.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
+from odoo.exceptions import UserError
+from odoo.tools.float_utils import float_compare, float_is_zero
+from dateutil.relativedelta import relativedelta
+
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Properties"
+    
+    _sql_constraints = [
+        ('check_prices', 'CHECK(expected_price > 0 AND selling_price >= 0)',
+         "Prices should be positive. Additionnaly, expected price can't be 0.")
+    ]
+    
+    # Metadata
+    name = fields.Char('Title', required=True, translate=True, default="Best House In Town")
+    active = fields.Boolean('Active', default=True)
+    description = fields.Text('Description', default="Yo yo yo, no dup for date and status !")
+    postcode = fields.Char('Postcode', translate=True)
+    property_type_id = fields.Many2one('estate.property.type', string='Property Type')
+    property_tag_ids = fields.Many2many('estate.property.tag', string='Property Tags')
+    property_offer_ids = fields.One2many('estate.property.offer', 'property_id')    
+    
+    # House Data
+    bedrooms = fields.Integer('Bedrooms', default=2)
+    living_area = fields.Integer('Living Area (sqm)')
+    facades = fields.Integer('Facades')
+    garage = fields.Boolean('Garage', default=False)
+    garden = fields.Boolean('Garden', default=False)
+    garden_area = fields.Integer('Garden Area (sqm)')
+    garden_orientation = fields.Selection(
+        string='Garden Orientation', 
+        selection = [('north', "North"), ('south', 'South'), ('east', 'East'), ('west', 'West')]
```

Comment:
```suggestion
        selection = [
            ('north', "North"),
            ('south', 'South'),
            ('east', 'East'),
            ('west', 'West'),
        ]
```
This is better. We can add values without breaking the git history.